ontobio/ontol.py

```diff
@@ -598,12 +598,49 @@ def logical_definitions(self, nid):
         else:
             return []
 
+    def definition(self, nid):
+        """
+        Text definition object for a node
+
+        Arguments
+        ---------
+        nid : str
+            Node identifier for entity to be queried
+
+        Return
+        ------
+        dict
+            definition object, dict(val=TEXT, xrefs=LIST)
+        """
+        return self._get_meta_prop(nid, 'definition')
+        
+    def definition_val(self, nid):
```

Comment:
This seems like this should just be a method on whatever is returned from `self.definition`. So if you want the string version, you would just say `str(definition(id))` instead of providing a separate method here.